samples/Client/DaprClient/Program.cs

```diff
@@ -3,353 +3,42 @@
-            Console.WriteLine("Returned: id:{0} | Balance:{1}", a.Id, a.Balance);
-
-            Console.WriteLine("Completed");
-        }
 
-        /// <summary>
-        /// This example shows how to invoke a POST method on a service listening on http.
-        /// Example:  curl -X POST http://127.0.0.1:5000/deposit -H "Content-Type: application/json" -d "{ \"id\": \"17\", \"amount\": 120 }"
-        /// </summary>
-        /// <param name="client"></param>
-        /// <remarks>
-        /// Before invoking this method, please first run RoutingSample.
-        /// </remarks>
-        /// <returns></returns>
-        internal static async Task InvokeWithdrawServiceOperationAsync(DaprClient client)
-        {
-            Console.WriteLine("Invoking withdraw");
-            var data = new { id = "17", amount = (decimal)10, };
-
-            // Invokes a POST method named "Withdraw" that takes input of type "Transaction" as define in the RoutingSample.            
-            await client.InvokeMethodAsync<object>("routing", "Withdraw", data, HttpInvocationOptions.UsingPost());
-
-            Console.WriteLine("Completed");
-        }
-
-        /// <summary>
-        /// This example shows how to invoke a GET method on a service listening on http.
-        /// Example:  curl -X GET http://127.0.0.1:5000/17
-        /// </summary>
-        /// <param name="client"></param>
-        /// <remarks>
-        /// Before invoking this method, please first run RoutingSample.
-        /// </remarks>
-        /// <returns></returns>
-        internal static async Task InvokeBalanceServiceOperationAsync(DaprClient client)
-        {
-            Console.WriteLine("Invoking balance");
-
-            // Invokes a GET method named "hello" that takes input of type "MyData" and returns a string.
-            var res = await client.InvokeMethodAsync<Account>("routing", "17", HttpInvocationOptions.UsingGet());
-
-            Console.WriteLine($"Received balance {res.Balance}");
-        }
-
-        internal static async Task InvokeThrowExceptionOperationAsync(DaprClient client)
-        {
-            Console.WriteLine("Invoking ThrowException");
-            var data = new { id = "17", amount = (decimal)10, };
-
-            try
-            {
-                // Invokes a POST method named "throwException" that takes input of type "Transaction" as defined in the ControllerSample.            
-                await client.InvokeMethodAsync("controller", "throwException", data, HttpInvocationOptions.UsingPost());
-            }
-            catch (RpcException ex)
+            Console.WriteLine("Hello, please choose a sample to run:");
+            for (var i = 0; i < Examples.Length; i++)
             {
-                var entry = ex.Trailers.Get(grpcStatusDetails);
-                var status = Google.Rpc.Status.Parser.ParseFrom(entry.ValueBytes);
-                Console.WriteLine("Grpc Exception Message: " + status.Message);
```

Comment:
The sample code to handle grpc exceptions got removed

Reply:
This sample was added before we had support in the box for automatically unpacking the status details. This was to document the workaround. Now we have this support in the box, and it's actually not meaningful anymore because we're getting rid of the gRPC -> HTTP invoke path wholesale.